docs/index.rst

```diff
@@ -2,6 +2,11 @@
 Qiskit Research documentation
 =============================
 
+.. toctree:: 
```

Comment:
What does this do? Does it matter that there is already another `toctree` directive below?

Reply:
This toctree adds an entry only visible in the left side menu called Home. The toctree below continues working and doing the same as before (displaying the content in the left side menu and in the page).

Reply:
Ok.